file.go

```diff
@@ -151,14 +151,14 @@ func MakeGitHubZipFileRequest(gitHubCommit GitHubCommit, gitHubToken string, ins
 
 	// This represents either a commit, branch, or git tag
 	var gitRef string
-	if gitHubCommit.GitRef != "" {
-		gitRef = gitHubCommit.GitRef
-	} else if gitHubCommit.CommitSha != "" {
+	if gitHubCommit.CommitSha != "" {
```

Comment:
Update the comment above this if-statement to explicitly state that order matters here! You can link to this bug as an example of why.

Reply:
@pete0emerson leaving this one for you

Reply:
@pete0emerson Do you want to make this change so we can wrap this PR up?

Reply:
Addressed in e088d3b